lib/pbench/cli/agent/commands/results/clear.py

```diff
@@ -0,0 +1,50 @@
+"""
+pbench-clear-results
+"""
+import logging
+import shutil
+import sys
+
+import click
+
+from pbench.agent.utils import setup_logging
+from pbench.cli.agent import pass_cli_context
+from pbench.agent.base import BaseCommand
+from pbench.cli.agent.options import common_options
+
+LOG = logging.getLogger(__name__)
+
+
+class ResultsClear(BaseCommand):
+    """Clear files and directories other than stuff that start
+       with tmp and start with tools
+    """
+
+    def __init__(self, context):
+        super().__init__(context)
+
+        setup_logging(debug=False, logfile=None)
+
+    def execute(self):
+        try:
+            for path in self.pbench_run.iterdir():
+                # NOTE WELL: we remove each entry explicitly using the full
+                # pbench_run path just in case some bug inadvertently places
+                # us in the wrong directory.
+                if not str(path.name).startswith(("tmp", "tools")):
+                    if path.is_dir():
+                        shutil.rmtree(path)
+                    else:
+                        path.unlink()
+            return 0
+        except Exception:
+            LOG.warn("Unable to cleanup the '%s' directory", self.pbench_run)
```

Comment:
Perhaps include the string representation of the exception?  Something like: `LOG.warn("Unable to clean up the '%s' directory, '%s'", self.pbench_run, exc)`?

Reply:
done